language-server/modules/langserver-core/src/test/resources/codeaction/make-type-public/config/convert_to_public_record_config3.json

```diff
@@ -0,0 +1,24 @@
+{
+  "line": 0,
+  "character": 38,
+  "expected": [
+    {
+      "title": "Convert 'DefaultModRecord3' to public",
+      "edits": [
+        {
```

Comment:
Here, we have to assert the file URI as well since the edited construct is in a different file than the current one.

Reply:
Done